web/src/components/pages/datasets/dataset-corpus-download.tsx

```diff
@@ -4,112 +4,133 @@ import {
-        </LabeledSelect>
-
-        <LabeledSelect
-          label={getString('language')}
-          name="bundleLocale"
-          value={locale}
-          onChange={handleLangChange}>
-          {sortedLocales.map(locale => (
-            <Localized key={locale} id={locale}>
-              <option value={locale} />
-            </Localized>
-          ))}
-        </LabeledSelect>
+      <div className="dataset-corpus-download-container">
+        <div className="table-text">
+          <PageHeading>
+            <Localized id="download-dataset-header" />
+          </PageHeading>
+          <Localized
+            id="download-delta-explainer"
+            elems={{
+              deltaLink: <DeltaReadMoreLink className="link" />,
+            }}>
+            <div />
+          </Localized>
+          <p style={{ marginTop: '2rem' }}>
+            <Localized id="download-dataset-tag" />
+          </p>
+        </div>
+        <div className="input-row">
+          <LabeledSelect
+            label={getString('language')}
+            name="bundleLocale"
+            value={locale}
+            onChange={handleLangChange}>
+            {useSortedLocales(
+              languagesWithDatasets.map(s => s.name),
+              getString
+            )[0].map(val => (
+              <Localized key={val} id={val}>
+                <option value={val} />
+              </Localized>
+            ))}
+          </LabeledSelect>
+        </div>
+        <div
+          style={{
+            display: 'flex',
+            width: '100%',
+            alignItems: 'start',
+            flexDirection: 'column',
+          }}>
+          {isLoading && <Spinner />}
+          {!isLoading && LanguageDatasets && (
+            <DatasetCorpusDownloadTable
+              onRowSelect={(selectedId: number) =>
+                setSelectedDataset(
+                  LanguageDatasets.find(d => d.id === selectedId)
+                )
+              }
+              releaseData={LanguageDatasets}
+              selectedId={selectedDataset?.id || LanguageDatasets[0].id}
+            />
+          )}
 
-        <ul className="facts">
-          <DatasetCorpusDownloadStats
-            releaseData={releaseData}
-            bundleState={bundleState}
-          />
-        </ul>
-        <DatasetDownloadEmailPrompt
-          urlPattern={
-            releaseData.bundleURLTemplate
-              ? releaseData.bundleURLTemplate
-              : releaseData.bundleUrl
-          }
-          bundleState={bundleState}
-        />
+          {selectedDataset && selectedDataset.download_path && (
+            <>
+              <DatasetDownloadEmailPrompt
+                selectedLocale={locale}
+                downloadPath={selectedDataset.download_path}
+                releaseId={selectedDataset.id.toString()}
+                checksum={selectedDataset.checksum}
+                size={formatBytes(selectedDataset.size, locale)}
+              />
+            </>
```

Comment:
The Fragment wrapper (`<></>`) here can be removed because it is wrapping only one JSX element.